support-frontend/assets/pages/digital-subscription-landing/components/digitalSubscriptionLandingHeader.jsx

```diff
@@ -214,16 +211,6 @@ function DigitalSubscriptionLandingHeader(props: PropTypes) {
         }
         <div className="digital-subscription-landing-header__cta" />
       </LeftMarginSection>
-      <HeroHanger>
-        <AnchorButton
-          aria-label="See Subscription options for Digital Subscription"
-          onClick={sendTrackingEventsOnClick('options_cta_click', 'DigitalPack', null)}
```

Comment:
Are we losing this tracking completely?

Reply:
The design excludes the big yellow button that used to be in this panel, so yes we won't have this tracking anymore. However, there is still tracking attached to the link clicks in the product cards, so we'll still get information about whether the use clicks through etc.